packages/data-stores/package.json

```diff
@@ -37,6 +37,7 @@
 		"@wordpress/api-fetch": "^3.21.1",
 		"@wordpress/data-controls": "^1.20.1",
 		"@wordpress/deprecated": "^2.11.0",
+		"@automattic/domain-picker": "^1.0.0-alpha.0",
```

Comment:
Is this change still necessary?

Reply:
Thanks! Removed in https://github.com/Automattic/wp-calypso/pull/50761/commits/d830dd88d6a08f61393a8e01e2fa33a92be7916b.